.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,51 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug", "needs-triage"]
+issue_body: false
+body:
+- type: markdown
+  attributes:
+    value: |
+      If you need help with running Minecraft, please visit us [on our Discord](https://discord.gg/multimc) before making a bug report.
+
+      Before submitting a bug report, please make sure you have read this *entire* form, and that:
+      * You have read the [FAQ](https://github.com/MultiMC/MultiMC5/wiki/FAQ) and it has not answered your question
+      * Your bug is not caused by Minecraft or any mods you have installed.
+      * Your issue has not been reported before, [make sure to use the search function!](https://github.com/MultiMC/MultiMC5/issues)
+
+      Do not forget to give your issue a descriptive title. "Bug in the instance screen" makes it hard to distinguish issues at a glance.
+- type: dropdown
+  attributes:
+    label: Operating System
+    description: If you know this bug occurs on multiple operating systems, select all you have tested.
+    multiple: true
+    options:
+    - Windows
+    - macOS
+    - "*nix"
```

Comment:
This should be `Linux`. No other unix-like OS is supported.

We may want to add Other, just in case, but the likelihood of those issues of being addressed is near zero.